Boyadjie/Uncover
# feat(form): created contact form

## 🤔 Why?

For our MVP, we want to have a contact form if the users want to share a feedback, report a bug ect...


## 💻 How?

I created a page, a form and inputs

## ✅ How to validate this PR?

- Pull and check if every inputs work well
- Check that the form is working as expected and contain all the required inputs
